service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -148,8 +149,13 @@ public StepResult doStep(FlightContext flightContext)
     }
 
     // launch the submission
-    RunSetStateResponse runSetStateResponse =
-        cbasService.createRunSet(cbasUri, samService.getTspsServiceAccountToken(), runSetRequest);
+    RunSetStateResponse runSetStateResponse;
+    try {
+      runSetStateResponse =
+          cbasService.createRunSet(cbasUri, samService.getTspsServiceAccountToken(), runSetRequest);
+    } catch (CbasServiceApiException e) {
+      throw new RetryException("Failed to submit run set. Will retry.", e);
```

Comment:
i think we should lower the number of step retries in our retry rule, maybe 2 retries with a bigger back off for now?